### DISCUSSION TITLE
Parameterized Quarto & Publishing to Posit Connect

### DISCUSSION THREAD

**Discussion by Ifitrains** [quarto-dev/quarto-cli#8227] [Q&A] [answered]
### Description

Is this feature https://docs.posit.co/connect/user/param-rmarkdown/ also planned for Quarto?

**Comment by mcanouil** [comment]
You mean <https://quarto.org/docs/computations/parameters.html>?
It's there since the start.

What holds the future is an homogenous framework agnostic to the engine knitr/Jupyter, but that's really a future thing.

**Reply by cderv** [reply, marked as answer]
I think the question is about support for publishing parametrized repo in Publish Connect with same kind of feature and interface for the published report. 

As explained in [docs.posit.co/connect/user/param-rmarkdown](https://docs.posit.co/connect/user/param-rmarkdown/)

To my knowledge I don't think this is yet something supported.

**Reply by mcanouil** [reply]
I am not sure the integration in Posit Connect will happen before it can be used also for Jupyter.

**Reply by Ifitrains** [reply]
okay thank you for your answers!
